src/edfi_amt_data_lake/api/changeVersion.py

```diff
@@ -29,6 +35,15 @@ def get_change_version_values_from_file(file) -> ChangeVersionValues:
     return ChangeVersionValues('0', '0')
 
 
+def _delete_downloaded_files() -> None:
+    import shutil
+    path = f"{config('CHANGE_VERSION_FILEPATH')}"
+    if not os.path.exists(path):
+        raise Exception("Directory not found!")
+    shutil.rmtree(path)
```

Comment:
This function is not working. It throws an exception:
`OSError: [WinError 145] The directory is not empty:`

Reply:
Done.